src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -161,12 +167,10 @@ export const NavBar: React.FC<NavBarPropsType> = ({
     enterVenue(parentVenueId, { customOpenRelativeUrl: openUrlUsingRouter });
   }, [parentVenueId, openUrlUsingRouter]);
 
+  const homeVenueId = sovereignVenue?.id ?? parentVenueId;
```

Comment:
If you use `useSovereignVenueId`, you won't need this as it will get either sovereignVenueId, parentVenueId or venueId

Reply:
I could not delete this part as `useSovereignVenueId` can be sometimes undefined but I used `useSovereignVenueId`

Reply:
parentVenueId could also be undefined presumably. Not every venue is going to have a parent. That’s an edgecase we probably need to explicitly handle here, not just with a default.